<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>校园场景人群检测与计数系统</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate_num_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>计算人数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_image_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>加载图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="notice_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>请将需要进行计数的图像放在/data的相应目录下，并重命名</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QFrame" name="restaurant_frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>491</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>50</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="res_result_1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="res_image_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="restaurant_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>食堂</string>
     </property>
    </widget>
    <widget class="QLabel" name="res_image_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="res_result_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="res_image_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="res_result_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>300</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="res_image_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="res_result_4">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="res_image_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="res_result_5">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>190</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="res_image_6">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>260</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="res_result_6">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>300</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QFrame" name="outdoor_frame">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>321</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>50</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="out_result_1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="out_image_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="outdoor_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>室外</string>
     </property>
    </widget>
    <widget class="QLabel" name="out_image_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="out_result_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="out_image_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>260</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="out_result_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="restaurant_frame_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>831</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>50</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="class_result_1">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="class_image_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="classroom_label">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>教室</string>
     </property>
    </widget>
    <widget class="QLabel" name="class_image_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="class_result_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="class_image_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="class_result_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="class_image_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>150</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="class_result_4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>190</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="class_image_5">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>40</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="class_result_5">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="class_image_6">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>150</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请单击“加载图像”</string>
     </property>
    </widget>
    <widget class="QLabel" name="class_result_6">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>190</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="notice_label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>780</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>由于计数功能使用多种深度学习算法，速度较慢，请耐心等待15-30s</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <zorder>restaurant_frame</zorder>
   <zorder>title</zorder>
   <zorder>calculate_num_button</zorder>
   <zorder>show_image_button</zorder>
   <zorder>notice_label</zorder>
   <zorder>clear_button</zorder>
   <zorder>outdoor_frame</zorder>
   <zorder>restaurant_frame_2</zorder>
   <zorder>notice_label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
